packages/electron-updater/package.json

```diff
@@ -16,6 +16,7 @@
     "out"
   ],
   "dependencies": {
+    "builder-util": "workspace:*",
```

Comment:
I don't think we're able to import `builder-util` into electron-updater as it is meant for local building. I think `retry` may need to be moved into `builder-util-runtime` in order for this PR to proceed.

Reply:
I moved the core retry function to builder-util-runtime, but exported a wrapper function of `retry` in `builder-util` that adds the original logging behavior, since the logging module remains in `builder-util`.